src/gauge/radial-gauge/radial-gauge.component.ts

```diff
@@ -0,0 +1,266 @@
+import { Component, AfterViewInit, ViewEncapsulation, 
+  ChangeDetectionStrategy, Input, Output, EventEmitter } from '@angular/core';
+
+import { ScaleLinear, scaleLinear } from 'd3-scale';
+import { line, curveLinear, Line } from 'd3-shape';
+
+import { BaseChartComponent } from './../../common/base-chart.component';
+import { ColorHelper } from '../../common/color.helper';
+
+@Component({
+  selector: 'ngx-charts-radial-gauge',
+  templateUrl: './radial-gauge.component.html',
+  styleUrls: ['./radial-gauge.component.css'],
```

Comment:
If you need to define a custom file I think it should be a `scss` file as the others are all Sass `scss` based files.

I think you want the base component `scss` file like the other components. So if you need a custom `scss` it would be:

```js
  styleUrls: [
    '../../common/base-chart.component.scss',
    './radial-gauge.component.scss'
  ],
```

otherwise with just the base component style:
```
styleUrls: ['../../common/base-chart.component.scss']
```